webpack.config.js

```diff
@@ -9,12 +11,19 @@ module.exports = {
   // Set the webpack4 mode 'none' for compatibility with the behavior
   // of the webpack3 bundling step.
   mode: 'none',
-  entry: './src/main.js',
+  entry: {
+    'addons-linter': './src/main.js',
```

Comment:
nit, would you mind to move this after the `...glob.sync(...).reduce(...)`?
just to be on the "safe side" and be absolutely sure that there is no change for it to be overwritten by the computed ones by mistake.